src/components/App/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { Header } from '../Header/Header';
+import { Button } from '../Button/Button';
+import { PagesRouter } from '..//../routes/PagesRouter';
+
+export const App = () => {
+  return (
+    <BrowserRouter>
+      <>
+        <Header />
+        <PagesRouter />
+        <Button />
```

Comment:
Remove <Button> component and put it inside every page, because it is not shared component and it will have different business logic for every page

Reply:
I have to share the Route in my components i.e. People, Planets, Starships?